articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+
+Technology is growing at a fast pace in recent years. Due to this, it is now easy to get lost in the rumpus of technological terms that keep popping up every day. Machine learning and deep learning are among the fast-growing technologies. Besides this two, [Internet of Things (IoT](https://internetofthingsagenda.techtarget.com/definition/Internet-of-Things-IoT)), [blockchain](https://en.wikipedia.org/wiki/Blockchain), and [Artificial Intelligence](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp) (AI) are among the fast-growing technologies.
```

Comment:
Tech Review: Calling machine learning and deep learning technologies might not be suitable. These are concepts over which various technologies and frameworks have been built. (We can instead emphasize on how most of the problems are being solved using machine learning and deep learning)

Reply:
noted and fixed.